app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
```

Comment:
Think how you can evaluate this more efficiently. Hint: accessing a value in dictionary by key gives you best algorithmic complexity

Reply:
when you use `not in` it means that you iterate through all the keys in dictionary until you find a match. When you access an element by a key it is instant match. Please change